llvm/include/llvm/BinaryFormat/ELFRelocs/RISCV.def

```diff
@@ -58,3 +58,5 @@ ELF_RELOC(R_RISCV_SET16,             55)
 ELF_RELOC(R_RISCV_SET32,             56)
 ELF_RELOC(R_RISCV_32_PCREL,          57)
 ELF_RELOC(R_RISCV_IRELATIVE,         58)
+ELF_RELOC(R_RISCV_CVPCREL_UI12,     224)
```

Comment:
This needs a comment noting that custom relocations start here. And I'm assuming this is still a "CORE-V only" compiler?

Reply:
Do you mean "CORE-V only" as in "only supports the one custom target CORE-V"? Then yes (I think).